src/hooks/useKnockoutExplorer.ts

```diff
@@ -108,31 +105,19 @@ async function configureFabric(): Promise<Explorer> {
 
         switch (data.type) {
```

Comment:
Let's add support for version property

switch (`${data.type}_${data.version}`) {
    case "initialize_": {
        const fabricDatabaseConnectionInfo: FabricDatabaseConnectionInfo = {
            endpoint: data.message.endpoint,
            databaseId: data.message.databaseId,
            resourceTokens: data.message.resourceTokens as { [resourceId: string]: string },
            resourceTokensTimestamp: data.message.resourceTokensTimestamp,
        };
        explorer = await createExplorerFabric(fabricDatabaseConnectionInfo);
        resolve(explorer);

        explorer.refreshAllDatabases().then(() => {
            openFirstContainer(explorer, fabricDatabaseConnectionInfo.databaseId);
        });
        scheduleRefreshDatabaseResourceToken();
        break;
    }
    case "initialize_ fabric2": {
        explorer = createExplorerFabric(data.message);
        await scheduleRefreshDatabaseResourceToken(true);
        resolve(explorer);
        await explorer.refreshAllDatabases();
        openFirstContainer(explorer, userContext.fabricContext.databaseConnectionInfo.databaseId);
        break;
    }

Reply:
Added a new message and explictly keep track of the message versions.